specs/operator/pre-upgrade.spec

```diff
@@ -0,0 +1,98 @@
+PIPELINES-18
```

Comment:
@ppitonak shall I remove IDs from these scenarios as we do not consider this as test cases?

Reply:
I'm not sure, I would leave it there as it won't hurt

Reply:
Hmm, this is part of the upgrade tests, let's keep it.